service/init.go

```diff
@@ -338,9 +362,34 @@ func (pcf *PCF) registerNF() {
 				//NRF Registration Successful, Trigger for UDR Discovery
 				pcf.discoverUdr()
 			}
-		}
+		} else {
+            initLog.Infof("PCF is not having Minimum Config to Register/Update to NRF")
```

Comment:
this is where keep alive would be stopped ?

Reply:
yes, before sending deregister we should stop timer